extension/opampextension/testdata/config.yaml

```diff
@@ -1,2 +1,4 @@
-endpoint: wss://127.0.0.1:4320/v1/opamp
+server:
+  ws:
```

Comment:
@tigrannajaryan @evan-bradley perhaps the supervisor should also make this separation in server transport configuration?

Reply:
I think that makes sense. In the future may want to move the handling for some of this configuration into a shared internal module, but this is a good first step.